app/Helpers/Geocoder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Log;
+
+/**
+ * Google Geocoder interface.
+ */
+class Geocoder
+{
+
+    protected $apikey = "AIzaSyDb1_XdeHbwLg-5Rr3EOHgutZfqaRp8THE";
+
+    /**
+     * Geocode an address.
+     *
+     * @return the latitude and longitude
+     */
+    public function geocode($location)
+    {
+        $url = "https://maps.googleapis.com/maps/api/geocode/json?address=";
+        $url .= urlencode($location.',United Kingdom');
```

Comment:
Are geocoder lookups always in the UK?

Reply:
Hmm no, that's incorrect.  They could be anywhere in the world.